MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: Validation in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the validation support in Syncfusion .NET MAUI DataForm (SfDataForm) and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Validation in .NET MAUI DataForm (SfDataForm)
+
+The data form validates the data and displays hints in the case of validation is not passed. In case of invalid data, the error message is shown at the bottom of the editor.
+
+## Built in validations
+
+The supported built in validations are as follows:
+
+* Data annotation
+
+### Data annotations
+
+You can validate the data using data annotation attributes.
+
+The numeric type like Int, Double, Decimal properties can be validated using the [Range](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.rangeattribute?view=netframework-4.8) attribute.
```

Comment:
remove Range annotation, as we not included numeric editor support.

Reply:
removed Range annotation